ingest/src/main/scala/geotrellis/osme/ingest/GenerateHistogram.scala

```diff
@@ -0,0 +1,27 @@
+package geotrellis.osme.ingest
+
+import geotrellis.raster._
+import geotrellis.raster.histogram._
+import geotrellis.spark._
+import geotrellis.spark.io._
+import geotrellis.spark.io.s3._
+import geotrellis.vector._
+import geotrellis.spark.util.SparkUtils
+import org.apache.spark.SparkConf
+import org.apache.spark.rdd.RDD
+
+object GenerateHistogram {
```

Comment:
Could you add a comment here about what this does and why you needed to do this?
If you can include a sample of a the `spark-submit` commend you used to run it, it would be great reference material.